Pod/API/Internal/Parse.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  Parse.swift
+//  ElJayKit
+//
+//  Created by Isaac Greenspan on 12/12/15.
+//  Copyright © 2015 Isaac Greenspan. All rights reserved.
+//
+
+import Foundation
+import AlamofireXMLRPC
+
+protocol XMLRPCDeserializable {
+    init(xmlrpcNode value: XMLRPCNode) throws
+}
+
+extension API {
+    enum ParseError: ErrorType {
+        case BadStructure
+        case BadStructureInArray
+    }
+    
+    /**
+     Internal XMLRPC call handler with single-item response parsing.
+     
+     - parameter method:              The method to call
+     - parameter parameters:          The parameters of the call, if any
+     - parameter responseType:        The type of item expected in the response
+     - parameter responsePostProcess: A function to apply to the parsed response before invoking the callback
+     - parameter callback:            The callback to execute with the result of the call
+     */
+    func execute<ResponseType: XMLRPCDeserializable>(
+        method: Method,
+        parameters: [String: Any]? = nil,
+        responseType: ResponseType.Type,
```

Comment:
It's what ties down the generic `ResponseType` type.  I guess it could be `responseType _: ResponseType.Type` to omit the parameter variable, but I'm not sure I see the point in that.  Is there some way to fulfill the generic without that parameter?